### PR TITLE
update style and data to persist on collapsing

### PR DESCRIPTION
**Pull request by JeanLoureiro**

### CODE REVIEW

**Comment by JeanLoureiro** [comment]
to pass the React Components dumb component rule, we have to update this component. At the moment it only fetches from timeseries fullnames.

File: src/DataServices/DataServices.ts [hunk 1]
```diff
@@ -122,6 +122,39 @@ const fetchTimeseriesByGroup = (dataSources: DataSource[], token: string) => {
   return forkJoin(requests).pipe(tap((ts) => console.log('got group names', ts)));
 };
 
+/**
+ * /api/timeseries/{connectionId}/fullnames
+ * Gets a list of time series full-name identifiers.
+ * @param dataSources connectionId is required
+ * @param token
+ */
+const fetchTimeseriesfullNames = (dataSources: DataSource[], token: string, group: any) => {
```
Comment (FranzThomsen1089) [inline src/DataServices/DataServices.ts:131]: F

File: src/DataServices/types.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ interface DataSource {
   to?: string;
   sheetName?: string;
   connectionJobLog?: string;
+  group?: string;
```
Comment (FranzThomsen1089) [inline src/DataServices/types.ts:17]: Props here should really be described

**Review by FranzThomsen1089 — CHANGES_REQUESTED** [review]

File: src/DataServices/types.ts (hunk 2 above)
Comment (JeanLoureiro) [reply src/DataServices/types.ts:32]: updated

File: src/DataServices/DataServices.ts (hunk 1 above)
Comment (JeanLoureiro) [reply src/DataServices/DataServices.ts]: updated

**Review by FranzThomsen1089 — APPROVED** [review]

**Review by goneale — APPROVED** [review]

**Pull request merged**